### ISSUE TITLE
Please provide requirements.txt file(s) with pinned versions

### ISSUE THREAD

**Issue by la3lma** [danspeech/danspeech#13]
... it's somewhere between a pain and impossible to get things to run if it's unknown which combination of dependencies worked for you.

At the very least, provide the output of "pip freeze".

**Comment by Rasmusafj** [comment]
Hi, 

Thanks for reporting the issue. 

Will update the repo with versions of dependencies for my working environment later today when I have 10 minutes.

**Comment by Rasmusafj** [comment]
I added a conda yml file of my virtual environment to the danspeech_training repo and a small installation description in the readme.md. This environment also includes the danspeech package. See [danspeech_training repo](https://github.com/danspeech/danspeech_training). Hope this helps. :)

I am able to install danspeech with pip in a fresh virtual conda env with python==3.6.10 without any trouble. So for now, I think we'll keep the requirements.txt as is. I did however create a new issue for the annoying warning that comes when using a newer version of pytorch (#14). 

If installing danspeech with pip fails in your setup, then let me know with more specifics about your setup and I might be able to update the requirements.txt and bump the pip package. :) 

Best
Rasmus
